data/json/items/armor/combat_exoskeleton_armor.json

```diff
@@ -0,0 +1,995 @@
+[
+  {
+    "id": "combat_exoskeleton_armor_head_heavy",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "heavy combat exoskeleton head armor" },
+    "description": "This 2mm thick exoskeleton armor was designed when the testing of heavier versions concluded that mobility is actually very important on the battlefield, especially when you can still brush off anything short of a rifle round.",
+    "symbol": "[",
+    "price": 2000000,
+    "price_postapoc": 10000,
+    "volume": "3300 ml",
+    "weight": "3500 g",
+    "flags": [ "ABLATIVE_EXOSKELETON_HEAD", "STURDY", "CANT_WEAR" ],
+    "armor": [
+      {
+        "layers": [ "OUTER" ],
+        "covers": [ "head" ],
+        "encumbrance": 8,
+        "coverage": 100,
+        "material": [
+          { "type": "nylon", "thickness": 1, "covered_by_mat": 5 },
+          { "type": "plastic_pad", "thickness": 2 },
+          { "type": "kevlar_layered", "thickness": 4.4 },
+          { "type": "qt_steel", "thickness": 6, "covered_by_mat": 95 }
+        ]
+      },
+      {
+        "layers": [ "OUTER" ],
+        "covers": [ "mouth" ],
+        "encumbrance": 0,
+        "coverage": 100,
+        "material": [
+          { "type": "plastic_pad", "thickness": 2 },
+          { "type": "kevlar_layered", "thickness": 4.4 },
+          { "type": "qt_steel", "thickness": 6 }
+        ]
+      },
+      {
+        "layers": [ "OUTER" ],
+        "covers": [ "eyes" ],
+        "encumbrance": 0,
+        "coverage": 100,
+        "material": [ { "type": "lvl4ballisticglass", "thickness": 2 } ]
+      }
+    ],
+    "pocket_data": [
+      {
+        "pocket_type": "CONTAINER",
+        "volume_encumber_modifier": 0,
+        "max_contains_volume": "300 ml",
+        "max_contains_weight": "500 g",
+        "max_item_length": "14 cm",
+        "moves": 100,
+        "description": "Pocket for a flashlight attachment.",
+        "flag_restriction": [ "HELMET_HEAD_ATTACHMENT" ],
+        "transparent": true
+      }
+    ]
+  },
+  {
+    "id": "combat_exoskeleton_armor_torso_heavy",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "heavy combat exoskeleton torso armor" },
+    "description": "This 2mm thick exoskeleton armor was designed when the testing of heavier versions concluded that mobility is actually very important on the battlefield, especially when you can still brush off anything short of a rifle round.",
```

Comment:
The heavy version is able to stop a regular .308 round at 68 bullet protection, are you suggesting I raise it to 100-120? I figured anything above .308 would start being "anti-material" and thus not be what the armor is supposed to protect the user against.

The descriptions have been updated though.

Reply:
Outdated, the numbers look good. I would still change the description to something indicating that it can stop most small arms fire, "anything short of a rifle round" reads like it only protects against pistol/shot.